<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="12" passed="8" failed="4" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-24T16:56:06 WAT" name="Default suite" finished-at="2023-01-24T17:00:25 WAT" duration-ms="259134">
    <groups>
    </groups>
    <test started-at="2023-01-24T16:56:06 WAT" name="Default test" finished-at="2023-01-24T17:00:25 WAT" duration-ms="259134">
      <class name="BetterLifeTest.LogingPageTest">
        <test-method signature="Verify_Admin_Login_LogOut()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:56:06 WAT" name="Verify_Admin_Login_LogOut" description="This test case is to ensure admin can login and log out with valid username and password" finished-at="2023-01-24T16:56:22 WAT" duration-ms="16204" status="FAIL">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//button[normalize-space()='Approve Portfolios']"}
  (Session info: MicrosoftEdge=109.0.1518.55)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.3.0', revision: 'a4995e2c09*'
System info: host: 'ENE', ip: '192.168.77.240', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.14'
Driver info: org.openqa.selenium.edge.EdgeDriver
Command: [bf5b9a68d6198d5a53c47f1788dde3bc, findElement {using=xpath, value=//button[normalize-space()='Approve Portfolios']}]
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 109.0.1518.55, ms:edgeOptions: {debuggerAddress: localhost:61653}, msedge: {msedgedriverVersion: 109.0.1518.61 (354b36f96bab..., userDataDir: C:\Users\bglag\AppData\Loca...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:61653/devtoo..., se:cdpVersion: 109.0.1518.55, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: bf5b9a68d6198d5a53c47f1788dde3bc]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//button[normalize-space()='Approve Portfolios']"}
  (Session info: MicrosoftEdge=109.0.1518.55)
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.3.0', revision: 'a4995e2c09*'
System info: host: 'ENE', ip: '192.168.77.240', os.name: 'Windows 11', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.14'
Driver info: org.openqa.selenium.edge.EdgeDriver
Command: [bf5b9a68d6198d5a53c47f1788dde3bc, findElement {using=xpath, value=//button[normalize-space()='Approve Portfolios']}]
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 109.0.1518.55, ms:edgeOptions: {debuggerAddress: localhost:61653}, msedge: {msedgedriverVersion: 109.0.1518.61 (354b36f96bab..., userDataDir: C:\Users\bglag\AppData\Loca...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:61653/devtoo..., se:cdpVersion: 109.0.1518.55, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: bf5b9a68d6198d5a53c47f1788dde3bc
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:569)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:387)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:379)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:70)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:39)
at com.sun.proxy.$Proxy14.click(Unknown Source)
at BetterLifePages.LoginPage.AdminHomeButton(LoginPage.java:49)
at BetterLifeTest.LogingPageTest.Verify_Admin_Login_LogOut(LogingPageTest.java:20)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Admin_Login_LogOut -->
        <test-method signature="Verify_Admin_Login_With_Empty_UserName_And_Password_Field()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:56:22 WAT" name="Verify_Admin_Login_With_Empty_UserName_And_Password_Field" description="This test case is to ensure admin cannot login with empty username and password field" finished-at="2023-01-24T16:56:38 WAT" duration-ms="15381" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Admin_Login_With_Empty_UserName_And_Password_Field -->
        <test-method signature="Verify_Admin_Login_With_Invalid_UserName_Invalid_Password()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:56:38 WAT" name="Verify_Admin_Login_With_Invalid_UserName_Invalid_Password" description="This test case is to ensure admin cannot login with an invalid username and an Invalid password" finished-at="2023-01-24T16:56:53 WAT" duration-ms="15514" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Admin_Login_With_Invalid_UserName_Invalid_Password -->
        <test-method signature="Verify_Admin_Login_With_Invalid_UserName_Valid_Password()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:56:53 WAT" name="Verify_Admin_Login_With_Invalid_UserName_Valid_Password" description="This test case is to ensure admin cannot login with an invalid username and a valid password" finished-at="2023-01-24T16:57:10 WAT" duration-ms="17371" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Admin_Login_With_Invalid_UserName_Valid_Password -->
        <test-method signature="Verify_Admin_Login_With_Valid_UserName_Invalid_Password()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:57:10 WAT" name="Verify_Admin_Login_With_Valid_UserName_Invalid_Password" description="This test case is to ensure admin cannot login with a valid username and an invalid password" finished-at="2023-01-24T16:57:25 WAT" duration-ms="14292" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Admin_Login_With_Valid_UserName_Invalid_Password -->
        <test-method signature="Verify_Direct_Supervisor_Login_Logout()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:57:25 WAT" name="Verify_Direct_Supervisor_Login_Logout" description="This test case is to ensure Direct Program Supervisor can login and log out with valid username and password" finished-at="2023-01-24T16:57:43 WAT" duration-ms="18176" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://portfolio-assignment.babbangonaapps.com/supervisor/dashboard] but found [https://portfolio-assignment.babbangonaapps.com/login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://portfolio-assignment.babbangonaapps.com/supervisor/dashboard] but found [https://portfolio-assignment.babbangonaapps.com/login]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at BetterLifeTest.LogingPageTest.Verify_Direct_Supervisor_Login_Logout(LogingPageTest.java:67)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Direct_Supervisor_Login_Logout -->
        <test-method signature="Verify_In_Season_Advisor_Login_Logout()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:57:43 WAT" name="Verify_In_Season_Advisor_Login_Logout" description="This test case is to ensure In-season coach can login and log out with valid username and password" finished-at="2023-01-24T16:58:05 WAT" duration-ms="22205" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://portfolio-assignment.babbangonaapps.com/supervisor/dashboard] but found [https://portfolio-assignment.babbangonaapps.com/login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://portfolio-assignment.babbangonaapps.com/supervisor/dashboard] but found [https://portfolio-assignment.babbangonaapps.com/login]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at BetterLifeTest.LogingPageTest.Verify_In_Season_Advisor_Login_Logout(LogingPageTest.java:88)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_In_Season_Advisor_Login_Logout -->
        <test-method signature="Verify_Supervisor_Login_With_Empty_UserName_And_Password_Field()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:58:05 WAT" name="Verify_Supervisor_Login_With_Empty_UserName_And_Password_Field" description="This test case is to ensure supervisor cannot login with empty username and password field" finished-at="2023-01-24T16:58:46 WAT" duration-ms="41089" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Supervisor_Login_With_Empty_UserName_And_Password_Field -->
        <test-method signature="Verify_Supervisor_Login_With_Invalid_UserName_Invalid_Password()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:58:46 WAT" name="Verify_Supervisor_Login_With_Invalid_UserName_Invalid_Password" description="This test case is to ensure supervisor cannot login with an invalid username and an Invalid password" finished-at="2023-01-24T16:59:11 WAT" duration-ms="24974" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Supervisor_Login_With_Invalid_UserName_Invalid_Password -->
        <test-method signature="Verify_Supervisor_Login_With_Invalid_UserName_Valid_Password()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:59:11 WAT" name="Verify_Supervisor_Login_With_Invalid_UserName_Valid_Password" description="This test case is to ensure supervisor cannot login with an invalid username and a valid password" finished-at="2023-01-24T16:59:41 WAT" duration-ms="29661" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Supervisor_Login_With_Invalid_UserName_Valid_Password -->
        <test-method signature="Verify_Supervisor_Login_With_Valid_UserName_Invalid_Password()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T16:59:41 WAT" name="Verify_Supervisor_Login_With_Valid_UserName_Invalid_Password" description="This test case is to ensure supervisor cannot login with a valid username and an invalid password" finished-at="2023-01-24T17:00:06 WAT" duration-ms="24823" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_Supervisor_Login_With_Valid_UserName_Invalid_Password -->
        <test-method signature="Verify_TGE_Supervisor_Login_Logout()[pri:0, instance:BetterLifeTest.LogingPageTest@2a265ea9]" started-at="2023-01-24T17:00:06 WAT" name="Verify_TGE_Supervisor_Login_Logout" description="This test case is to ensure TGE supervisor can login and log out with valid username and password" finished-at="2023-01-24T17:00:25 WAT" duration-ms="19330" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://portfolio-assignment.babbangonaapps.com/supervisor/dashboard] but found [https://portfolio-assignment.babbangonaapps.com/login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://portfolio-assignment.babbangonaapps.com/supervisor/dashboard] but found [https://portfolio-assignment.babbangonaapps.com/login]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at BetterLifeTest.LogingPageTest.Verify_TGE_Supervisor_Login_Logout(LogingPageTest.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Verify_TGE_Supervisor_Login_Logout -->
      </class> <!-- BetterLifeTest.LogingPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
